app/controllers/account/tasks_controller.rb

```diff
@@ -36,12 +38,24 @@ def update
 
   def destroy
     resource.destroy
-    redirect_to account_workspace_project_path(parent.workspace_id, parent.id)
+    redirect_to account_workspace_project_path(parent.workspace_id, parent)
   end
 
   def move
-    resource.update_attributes(task_movement_params)
-    redirect_to account_workspace_project_path(parent.workspace_id, parent.id)
+    resource.update(task_movement_params)
+    redirect_to account_workspace_project_path(parent.workspace_id, parent)
+  end
+
+  def watch
+    @task = resource
+
+    if @task.users.include? current_user
```

Comment:
wrap include? in "( )"

Reply:
no need to use for class lvl methods: `attr_reader` `validate` that u usually call on class lvl
but for others - stick to the point of readability